src/Storage.Net/Storage.Net.csproj

```diff
@@ -7,14 +7,14 @@
       <GenerateDocumentationFile>true</GenerateDocumentationFile>
       <AssemblyName>Storage.Net</AssemblyName>
       <PackageId>Storage.Net</PackageId>
-      <PackageIconUrl>http://i.isolineltd.com/nuget/storage.png</PackageIconUrl>
+      <PackageIcon>storage.png</PackageIcon>
       <PackageProjectUrl>https://github.com/aloneguid/storage</PackageProjectUrl>
       <RepositoryUrl>https://github.com/aloneguid/storage</RepositoryUrl>
       <RepositoryType>GitHub</RepositoryType>
       <Version>4.0.1</Version>
       <FileVersion>4.0.0.0</FileVersion>
       <AssemblyVersion>4.0.0.0</AssemblyVersion>
-      <PackageLicenseUrl>https://github.com/aloneguid/storage/blob/master/LICENSE</PackageLicenseUrl>
+      <PackageLicense>https://github.com/aloneguid/storage/blob/master/LICENSE</PackageLicense>
       <Description>Storage.Net abstracts storage implementation like blobs, tables and messages from the .NET Application Developer. It's aimed to provide a generic interface regardless on which storage provider you are using. It also provides both synchronous and asynchronous alternatives of all methods and implements it to the best effort possible. Supports .NET 4.5.1 and .NET Standard 1.6.</Description>
```

Comment:
Check what the minimum .Net Standard has been used throughout solution.
Update description if higher than 1.6